app/static/tests/e2e/sensitivity.etest.ts

```diff
@@ -22,10 +22,11 @@ test.describe("Sensitivity tests", () => {
         // Edit settings
         await expect(await page.isVisible(".modal")).toBe(false);
         await page.click("#sensitivity-options .btn-primary");
-        await expect(await page.isVisible(".modal")).toBe(true);
-        await expect(await page.innerText(".modal-title")).toBe("Vary Parameter");
+        await expect(await page.locator("#edit-param .modal")).toBeVisible();
```

Comment:
This test was locating edit session modal instead of edit param but wasn't failing because it never ran since `test.only` was set.